samples/core/guide/autograph_control_flow.ipynb

```diff
@@ -0,0 +1,770 @@
+      },
+      "cell_type": "code",
+      "source": [
+        "! pip install tf-nightly\n",
+        "\n",
+        "from __future__ import division, print_function, absolute_import\n",
+        "\n",
+        "import tensorflow as tf\n",
+        "from tensorflow.contrib import autograph\n",
+        "\n",
+        "import matplotlib.pyplot as plt"
+      ],
+      "execution_count": 0,
+      "outputs": []
+    },
+    {
+      "metadata": {
+        "id": "Ry0TlspBZVvf",
+        "colab_type": "text"
+      },
+      "cell_type": "markdown",
+      "source": [
+        "Here's a quick example of how it works. Autograph can convert functions like this:"
+      ]
+    },
+    {
+      "metadata": {
+        "id": "aA3gOodCBkOw",
+        "colab_type": "code",
+        "colab": {}
+      },
+      "cell_type": "code",
+      "source": [
+        "def g(x):\n",
+        "  if x > 0:\n",
+        "    x = x * x\n",
+        "  else:\n",
+        "    x = 0.0\n",
+        "  return x"
+      ],
+      "execution_count": 0,
+      "outputs": []
+    },
+    {
+      "metadata": {
+        "id": "LICw4XQFZrhH",
+        "colab_type": "text"
+      },
+      "cell_type": "markdown",
+      "source": [
+        "Into graph-compatible functions like this:"
+      ]
+    },
+    {
+      "metadata": {
+        "id": "_EMhGUjRZoKQ",
+        "colab_type": "code",
+        "colab": {}
+      },
+      "cell_type": "code",
+      "source": [
+        "print(autograph.to_code(g))"
+      ],
+      "execution_count": 0,
+      "outputs": []
+    },
+    {
+      "metadata": {
+        "id": "xpK0m4TCvkJq",
+        "colab_type": "text"
+      },
+      "cell_type": "markdown",
+      "source": [
+        "You can take code written for eager execution and run it in graph mode. You get the same results, but with all the benfits of graphs:"
+      ]
+    },
+    {
+      "metadata": {
+        "id": "I1RtBvoKBxq5",
+        "colab_type": "code",
+        "colab": {}
+      },
+      "cell_type": "code",
+      "source": [
+        "print('Original value: %2.2f' % g(9.0))  "
+      ],
+      "execution_count": 0,
+      "outputs": []
+    },
+    {
+      "metadata": {
+        "id": "Fpk3MxVVv5gn",
+        "colab_type": "text"
+      },
+      "cell_type": "markdown",
+      "source": [
+        "Generate a graph-version and call it:"
+      ]
+    },
+    {
+      "metadata": {
+        "id": "SGjSq0WQvwGs",
+        "colab_type": "code",
+        "colab": {}
+      },
+      "cell_type": "code",
+      "source": [
+        "tf_g = autograph.to_graph(g)\n",
+        "\n",
+        "with tf.Graph().as_default():  \n",
+        "  # The result works like a regular op: takes tensors in, returns tensors.\n",
+        "  # You can inspect the graph using tf.get_default_graph().as_graph_def()\n",
+        "  g_ops = tf_g(tf.constant(9.0))\n",
+        "  with tf.Session() as sess:\n",
+        "    print('Autograph value: %2.2f\\n' % sess.run(g_ops))  "
```

Comment:
Alternatively, could be also "Tensor value", if we want to avoid overbranding (at any rate, should be consistent with the subsequent cells).

Reply:
Switched these to "Eager result" and "Graph result".